BoxContentSDK/BoxContentSDK/Requests/BOXFolderItemsRequest.m

```diff
@@ -95,6 +100,8 @@ - (void)performRequestWithCached:(BOXItemsBlock)cacheBlock refreshed:(BOXItemsBl
         NSMutableArray *results = [[NSMutableArray alloc] init];
 
         BOXItemsBlock localRefreshBlock = ^(NSArray *items, NSError *error){
+            self.paginatedRequest = nil;
```

Comment:
Well, I'm trying to avoid changing it too much, or too hastily.
When we create and set off a new paginated request we set it as the paginated request, and if we complete the existing one we nil our paginated request. Seems complete to me on handling each leaf in the tree.
What is your concern? Seem like we're missing anything?